opennlp-tools/src/main/java/opennlp/tools/cmdline/namefind/TokenNameFinderTrainerTool.java

```diff
@@ -121,23 +126,35 @@ public String getShortDescription() {
 
       for (File resourceFile : resourceFiles) {
         String resourceName = resourceFile.getName();
-        //gettting the serializer key from the element tag name
-        //if the element contains a dict attribute
+
+        // if the element contains a dict attribute we try gettting
+        // the serializer key from the element tag name
         for (Element xmlElement : elements) {
           String dictName = xmlElement.getAttribute("dict");
           if (dictName != null && dictName.equals(resourceName)) {
             serializer = artifactSerializers.get(xmlElement.getTagName());
           }
         }
-        // TODO: Do different? For now just ignore ....
-        if (serializer == null)
-          continue;
+
+        // otherwise we use the extension
+        if (Objects.isNull(serializer)) {
```

Comment:
not sure I would use that here, in the entire code base we wrote xyz == null or xyz != null